Blaise-Shyaka/politico-react
# [#172014391] Create reset password pages

# What does this PR do?

It creates pages allowing a user to reset a forgotten password

# Description of the tasks to be completed

- Create a page where a user submits an email. The email is used to send a reset password link
- Create a page where a user can write and confirm a newly created password

# How should this be manually tested?

- Clone this repository
- In the root folder, run `npm start` to view the app locally in the browser. A home page should be displayed by default
- Below the sign in form, click 'Forgot password'
- You should see a page prompting a user to put in an email. 
- Click the submit button 
- You should see a page prompting a user to create a new password and confirm it

# What are the relevant pivotal tracker stories?

#172014391

# Screenshots

![Screenshot (174)](https://user-images.githubusercontent.com/52489421/77981180-80248b80-7309-11ea-935b-6cb1030ecf5a.png)
![Screenshot (175)](https://user-images.githubusercontent.com/52489421/77981202-8b77b700-7309-11ea-842d-83350b99c24b.png)
![Screenshot (181)](https://user-images.githubusercontent.com/52489421/77981220-95011f00-7309-11ea-8e67-99498773506e.png)
![Screenshot (180)](https://user-images.githubusercontent.com/52489421/77981215-929ec500-7309-11ea-8bfc-5d9ae496b820.png)
![Screenshot (183)](https://user-images.githubusercontent.com/52489421/77981233-9df1f080-7309-11ea-8c57-068d3b2ddaf6.png)
![Screenshot (182)](https://user-images.githubusercontent.com/52489421/77981227-99c5d300-7309-11ea-9d60-4317dda1b7bb.png)
